test/unit/org/apache/cassandra/index/sai/cql/BM25Test.java

```diff
@@ -64,6 +64,22 @@ public void testTwoIndexes()
         assertRows(result, row(1));
     }
 
+    @Test
+    public void testDeletedRow()
+    {
+        // create un-analyzed index
+        createTable("CREATE TABLE %s (k int PRIMARY KEY, v text)");
+        // create analyzed index
+        analyzeIndex();
```

Comment:
Nit: I wonder if we could rename `analyzeIndex` to `createAnalyzedIndex` and get rid of the comment.